src/EFCore.Relational/Migrations/Internal/MigrationsModelDiffer.cs

```diff
@@ -209,7 +210,14 @@ public virtual IReadOnlyList<MigrationOperation> GetDifferences(IRelationalModel
                 }
                 else if (type == typeof(DropColumnOperation))
                 {
-                    dropColumnOperations.Add(operation);
+                    if (string.IsNullOrWhiteSpace(((ColumnOperation)operation).ComputedColumnSql))
+                    {
+                        dropColumnOperations.Add(operation);
+                    }
```

Comment:
Instead of storing this in the migrations operation, let's just use the DiffContext instead. The code here would be something like this:
```suggestion
                    if (string.IsNullOrWhiteSpace(diffContext.FindColumn((DropColumnOperation)operation).ComputedColumnSql)
                    {
                        dropColumnOperations.Add(operation);
                    }
```

Reply:
Also double-check the behavior of `null` vs whitespace. I think there's an overload of HasComputedColumnSql() that marks the column as computed, but doesn't actually specify the SQL. Although, it probably throws that it's not defined before it gets here...

Reply:
> Instead of storing this in the migrations operation, let's just use the DiffContext instead.

Done, check it out.

> Also double-check the behavior of null vs whitespace. I think there's an overload of HasComputedColumnSql() that marks the column as computed, but doesn't actually specify the SQL. Although, it probably throws that it's not defined before it gets here...

Seems so - I did a quick console test using this PR code, and got the following exception stack trace:

```
Unhandled exception. System.InvalidOperationException: The computed column SQL has not been specified for the column 'Name.Blogs'. Specify the SQL before using Entity Framework to create the database schema.
   at Microsoft.EntityFrameworkCore.Migrations.Internal.MigrationsModelDiffer.Initialize(ColumnOperation columnOperation, IColumn column, RelationalTypeMapping typeMapping, Boolean isNullable, IEnumerable`1 migrationsAnnotations, Boolean inline) in /home/roji/projects/efcore/src/EFCore.Relational/Migrations/Internal/MigrationsModelDiffer.cs:line 1145
   at Microsoft.EntityFrameworkCore.Migrations.Internal.MigrationsModelDiffer.Add(IColumn target, DiffContext diffContext, Boolean inline)+MoveNext() in /home/roji/projects/efcore/src/EFCore.Relational/Migrations/Internal/MigrationsModelDiffer.cs:line 1090
```

<details>
<summary>Test code</summary>

```c#
await using var ctx = new BlogContext();
await ctx.Database.EnsureDeletedAsync();
await ctx.Database.EnsureCreatedAsync();

public class BlogContext : DbContext
{
    public DbSet<Blog> Blogs { get; set; }

    static ILoggerFactory ContextLoggerFactory
        => LoggerFactory.Create(b => b.AddConsole().AddFilter("", LogLevel.Information));

    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
        => optionsBuilder
            .UseSqlServer(@"Server=localhost;Database=test;User=SA;Password=Abcd5678;Connect Timeout=60;ConnectRetryCount=0")
            .EnableSensitiveDataLogging()
            .UseLoggerFactory(ContextLoggerFactory);

    protected override void OnModelCreating(ModelBuilder modelBuilder)
        => modelBuilder.Entity<Blog>().Property(b => b.Name).HasComputedColumnSql();
}

public class Blog
{
    public int Id { get; set; }
    public string Name { get; set; }
}
```

</details>

Reply:
No, it's a valid model--we just can't create the full schema for it.

Reply:
Hmm, that seems... nuanced... But doesn't really matter.